BRAVE/docker-compose.prod.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.8"
+
+services:
+  django:
+    build: ./backend
+    container_name: dev-django
+    command: python manage.py runserver 0.0.0.0:8000
+    ports:
+      - 8001:8000
+    env_file:
+      - ./.env.dev
+      - ./backend/.env
+    volumes:
+      - ./backend/:/srv/app
+      - static:/srv/app/static
+      - media:/srv/app/media
+    depends_on:
+      - db
+  react-prod:
```

Comment:
leave service name as `react` and rename container to `prod-react` to be consistent with naming of other services and containers.